packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.types.ts

```diff
@@ -77,6 +78,23 @@ export interface IDetailsHeaderProps extends React.Props<DetailsHeaderBase>, IDe
   className?: string;
 }
 
+export interface IDetailsHeaderProps extends IDetailsHeaderBaseProps {

```

Comment:
Basically, `IDetailsHeaderProps` is now "Here's the props we'll give you in `onRender`" and `IDetailsHeaderBaseProps` is now "Here's the props the actual component requires".